tools/python/clean_strings_txt.py

```diff
@@ -396,8 +396,8 @@ def read_hardcoded_categories():
 
     if args.validate:
         if find_unused():
-            print(
-                "ERROR: there are unused strings\n(run \"{0} -s\" to delete them)\nTerminating...".format(prog_name))
+            print("ERROR: there are unused strings (run \"{0} -s\" to delete them)".format(prog_name))
+            print("If you're running generate_localizations.sh, add -u to generate anyway.\nExiting...")
```

Comment:
Looks hacky :)
Is it possible to check exit status != 0 in generate_localizations and print there?